openapi/components/schemas/SubscriptionCancellation.yaml

```
@@ -97,6 +97,17 @@ properties:
     description: Date and time when the subscription is deactivated.
     type: string
     format: date-time
+  churnTimePolicy:
+    description: |-
+      Specifies when the subscription is to be canceled.
```

```suggestion
      Specifies when the subscription is to be churned.
```
or
```suggestion
      Specifies when the subscription is to be deactivated.
```
based on the description of `churnTime` field